io/EptReader.cpp

```diff
@@ -121,32 +161,30 @@ void EptReader::initialize()
     m_queryBounds = m_args.bounds();
     handleOriginQuery();
 
-    const auto& scale(m_info->scale());
-    const auto& offset(m_info->offset());
-
     log()->get(LogLevel::Debug) << "Query bounds: " << m_queryBounds <<
         std::endl;
-    log()->get(LogLevel::Debug) << "Scale: " <<
-        scale[0] << ", " << scale[1] << ", " << scale[2] << std::endl;
-    log()->get(LogLevel::Debug) << "Offset: " <<
-        offset[0] << ", " << offset[1] << ", " << offset[2] << std::endl;
     log()->get(LogLevel::Debug) << "Threads: " << m_pool->size() << std::endl;
 }
 
 void EptReader::handleOriginQuery()
 {
     if (m_args.origin().empty()) return;
```

Comment:
Yes that's correct.  You're right, that's missing from the "spec" (I hesitate to call it a spec since it's just a pretty informal description) at this point.